### PR TITLE
One off bug fixes

### PR DESCRIPTION
**Pull request by kfinley-rbi**
- [x] appearance, delete css properties
- [x]  overflow - allowed values: 'hidden' or 'none', overflowY and overflowX are not allowed
- [x] whiteSpace - likely indicating the need for ellipsis
- [x]  textOverflow - ^^

### CODE REVIEW

File: codemods/styled-components-to-ucl.ts [hunk 1]
```diff
@@ -136,9 +135,8 @@ export function transformStyledCompoentsToUCL(
 
         logManualWork({
           filePath: fileInfo.path,
-          helpfulMessage: `The codemod to convert styled components to UCL nodes could not guarantee the correctness when encountering the variable for \`${
-            variable.id.name
-          }\`.
+          helpfulMessage: `The codemod to convert styled components to UCL nodes could not guarantee the correctness when encountering the variable for \`${variable.id.name
```
Comment (kfinley-rbi) [inline codemods/styled-components-to-ucl.ts:138]: preemptive prettier

File: codemods/utils/mappings.ts [hunk 2]
```diff
@@ -429,19 +472,6 @@ export const postToRNTransform = (identifier, value, needsFlexRemapping) => {
   };
 };
 
-const stylesToValue = (j: JSCodeshift, group, value) => {
```
Comment (kfinley-rbi) [inline codemods/utils/mappings.ts:432]: THis wasn't used

File: codemods/utils/parse-expression.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import { JSCodeshift } from "jscodeshift";
-import * as _ from "lodash/fp";
-import { brandFontMap, primitiveMap, valueToType } from "./mappings";
+import { JSCodeshift } from 'jscodeshift';
+import * as _ from 'lodash/fp';
+import { brandFontMap, primitiveMap, valueToType } from './mappings';
```
Comment (kfinley-rbi) [inline codemods/utils/parse-expression.ts:3]: These are all prettier changes

File: examples/__testfixtures__/styled-components-to-ucl/unsupported.output.ts [hunk 4]
```diff
@@ -1,9 +1,6 @@
 import { Box, FormControl, Input, Text } from '@rbilabs/universal-components';
 
-export const Scrollable = Box.withConfig({
-  // TODO: RN - unsupported CSS
-  // overflowX: 'auto',
-});
```
Comment (bkasten-rbi) [inline examples/__testfixtures__/styled-components-to-ucl/unsupported.output.ts:6]: do you think we should just default to commenting these out? I suspect that we may need to account for this overflow?

**Comment by bkasten-rbi** [comment]
better to move forward and adjust 👍

File: examples/__testfixtures__/styled-components-to-ucl/unsupported.output.ts (hunk 4 above)
Comment (kfinley-rbi) [reply examples/__testfixtures__/styled-components-to-ucl/unsupported.output.ts:6]: Will change

**Pull request merged**